storage/mysql/log_storage_test.go

```diff
@@ -17,8 +17,6 @@ import (
 	"github.com/google/trillian/testonly"
 )
 
-// TODO(al): add checking to all the Commit() calls in here.
```

Comment:
(style) FYI, there's a rule of thumb to avoid passing `*testing.T` objects to helper functions, but in this case I think the simpler code is worth it.